librz/core/cmd/cmd_print.c

```diff
@@ -1275,6 +1276,16 @@ static void cmd_print_fromage(RzCore *core, const char *input, const ut8 *data,
 			free(s);
 		}
 	} break;
+	case 'A': // "pFA"
+	{
+		char *s = rz_axml_decode(data, size);
+		if (s) {
+			rz_cons_printf("%s", s);
+			free(s);
+		} else {
+			eprintf("Malformed object: did you supply enough data?\ntry to change the block size (see b?)\n");
```

Comment:
RZ_LOG_ERROR

Reply:
I already said so in the previous PR, this is the same as all the other cases in this function. I don't plan to address this in this PR but rather just convert it when the `pF` commands will be converted to rzshell.